src/forms/assetForm.tsx

```diff
@@ -1075,41 +1107,47 @@ class AssetForm extends React.Component<AssetFormProps, AssetFormState> {
             </FormGroup>
           </Card>
           <Card>
-            <FormGroup label={AssetFormLabels.datacenter} inline={false}>
+            <FormGroup label={AssetFormLabels.site} inline={false}>
               <DatacenterSelect
                 popoverProps={{
                   minimal: true,
                   popoverClassName: "dropdown",
                   usePortal: true,
                 }}
-                items={this.getValidDatacenters()}
-                onItemSelect={(datacenter: DatacenterObject) => {
-                  this.state.currDatacenter
+                items={this.state.sites}
+                onItemSelect={(site: DatacenterObject) => {
+                  this.state.currSite
                     ? this.showChangeWarningAlert(
-                        "Are you sure you want to change datacenter? This will clear all datacenter related properties",
-                        datacenter
+                        "Are you sure you want to change site? This will clear all site related properties.",
+                        site
                       )
-                    : this.handleDatacenterSelect(datacenter);
+                    : this.handleSiteSelect(site);
                 }}
-                itemRenderer={renderDatacenterItem}
+                itemRenderer={renderSiteItem}
                 itemPredicate={filterDatacenter}
                 noResults={<MenuItem disabled={true} text="No results." />}
               >
                 <Button
                   rightIcon="caret-down"
                   text={
-                    this.state.currDatacenter && this.state.currDatacenter.name
-                      ? this.state.currDatacenter.name
-                      : "Select a Datacenter"
+                    this.state.currSite && this.state.currSite.name
+                      ? this.state.currSite.name
+                      : "Select a Site"
                   }
                 />
               </DatacenterSelect>
             </FormGroup>
           </Card>
 
-          {this.state.currDatacenter ? (
+          {this.state.currSite ? (
             <div>
+              {" "}
               <Card>{this.renderModelFields()}</Card>
+            </div>
+          ) : null}
+
+          {this.state.currSite && !this.state.currSite.is_storage ? (
```

Comment:
.... (2) if a site is selected AND it's a datacenter, then show rest of the options

Reply:
so we don't show racks / chassis / network ports / power ports for any asset going into an offline site